src/Utils/identityVerificationRequirements.ts

```diff
@@ -25,8 +25,8 @@ export const bidderNeedsIdentityVerification = ({
   bidder,
 }: {
   sale: IdentityVerificationRequireable
-  user: IdentityVerifiable | null
-  bidder: Bidder | null
+  user?: IdentityVerifiable
```

Comment:
What motivates this diff? This makes sense for bidder, as sometimes there is no bidder object within scope, but there's always a `me` for us to consider (even if that me is `null`). I'm probably just not understanding some nuance of TS here, but thanks for clarifying!

Reply:
Having both `value: Type | null` and `value?: Type` next to each other makes understanding the signature quite tricky. Also the `?` symbol aligns more with TS's new optional chaining `value?.attr` both semantically and visually.